qiskit/providers/aer/noise/errors/standard_errors.py

```diff
@@ -49,14 +52,21 @@ def kraus_error(noise_ops, standard_gates=True, canonical_kraus=False):
         raise NoiseError("Invalid Kraus error input.")
     if not noise_ops:
         raise NoiseError("Kraus error noise_ops must not be empty.")
+
+    if standard_gates:
```

Comment:
For these standard_gates kwargs that are being deprecated it would probably be better to change the default value to `None`, and then raise a deprecation warning if `standard_gates is not None`.